ansible_collections/arista/avd/roles/eos_designs/python_modules/base/avdstructuredconfig.py

```diff
@@ -174,7 +174,7 @@ def ipv6_unicast_routing(self):
         return None
 
     @cached_property
-    def ip_routing_ipv6_interfaces(self):
+    def ip_routing_ipv6_interfaces(self) -> bool | None:
```

Comment:
Thanks for taking the time to add these return types everywhere - could you also add it to router_multicast below line 186 and other you could have skipped so that things are aligned?

Reply:
Yes sure, Thanks.